core/templates/lru.h

```diff
@@ -106,7 +106,7 @@ class LRUCache {
 	void set_capacity(size_t p_capacity) {
 		if (capacity > 0) {
 			capacity = p_capacity;
-			while (_map.size() > capacity) {
+			while (_map.get_num_elements() > capacity) {
```

Comment:
ah my mistake, did not rename this to size() to conform to the other containers.